### ISSUE TITLE
YouTube HTTPS issue

### ISSUE THREAD

**Issue by gilbitron** [gilbitron/Nivo-Lightbox#61]
> I thought I'd mention a minor glitch I found in your javascript. I bought a site with your js in it, which lead me here.
> 
> nivo-lightbox.min.js
> 
> You had:
> 
> ... if(o[1]=="youtu"){a="http://www.youtube.com/embed ...
> 
> this section of code converts the short link into an embed link. While this works, it has a minor issue of ONLY working on http sites, as the protocall is hard coded.
> 
> Changing it to:
> 
> if(o[1]=="youtu"){a="https://www.youtube.com/embed
> 
> allows it to work on both http and https sites, as https content loads correctly over http, however the inverse is not true on all browsers.
> 
> There are two instances of the issue in your javascript.

**Comment by deidyomega** [comment]
Looks like there is already a pull request for this:

https://github.com/gilbitron/Nivo-Lightbox/pull/58/commits/752fc562142fb7661b4817523fd2275e9e39de12

**Comment by LukeTowers** [comment]
Yeah, I submitted that pull request a while ago. Is there anything I have to do on my end to push it through?

**Comment by LukeTowers** [comment]
Although, I might have to submit it again, the fix wasn't included in the minified version of the JS.

**Comment by alejo-jm** [comment]
hello the error with this is on the lines (197,201,205), this line have burned:
http://www. should be //www., for keep the https or http on site working

**Comment by LukeTowers** [comment]
Yes, that is what the pull request changes.

**Comment by tmugford** [comment]
This fix was merged in PR https://github.com/Codeinwp/Nivo-Lightbox-jQuery/pull/58.
